### DISCUSSION TITLE
关于 core 和 v1 两个 repo（的 release）的关系

### DISCUSSION THREAD

**Discussion by ouuan** [nxtrace/NTrace-core#187] [Q&A] [answered]
我就直接说我直接面临的问题吧。我在维护 https://aur.archlinux.org/packages/nexttrace-bin ，而我发现现在两个 repo 都会发相同的 release，我应该 follow 哪个来更新 AUR package 呢？另外我还观察到这两处 release 同一个版本号的 binary 内容都是不一致的，感觉这也会产生混淆。

**Comment by tsosunchia** [comment, marked as answer]
你好，首先特别感谢你帮助维护AUR源，

>我应该 follow 哪个来更新 AUR package 呢？

这个要看你的策略了，是稳健一点，还是更快一点，你可以再看一下项目README

>关于NTrace-V1和NTrace-core两个仓库的说明：
二者将大体上保持一致。所有的开发工作均在NTrace-V1仓库中进行。NTrace-V1仓库首先发布新版本，在稳定运行一段时间后（时长不定），我们会把版本同步至NTrace-core。这意味着NTrace-V1仓库充当了一个“测试版”的角色。
请注意，版本同步也存在例外。如果NTrace-V1的某个版本出现了严重的bug，NTrace-core会跳过这一有缺陷的版本，直接同步到下一个修复了该问题的版本。

关于你提到的

>我还观察到这两处 release 同一个版本号的 binary 内容都是不一致的

这个不应该呀，tag两个仓库都是指相向的同一个commit

**Comment by tsosunchia** [comment]
比如最新的TAG v1.2.4
core和V1仓库 指向的都是同一个commit   b2870a8

**Comment by ouuan** [comment]
这个是没做到 reproducible build，比如版本信息里有构建时间

**Reply by tsosunchia** [reply]
我好像知道你在说哪个了，你是不是在说 nexttrace -v的时候 显示的不一样？

**Reply by ouuan** [reply]
> 你是不是在说 nexttrace -v的时候 显示的不一样？

嗯

**Reply by tsosunchia** [reply]
因为俩仓库是分别action build的，不是同一个bin上传，所以会存在一些差别。
